examples/using-routes-todo-app/index.js

```diff
@@ -0,0 +1,82 @@
+const Tram = require('../../tram-one')
+const app = new Tram()
+
+const Todos = require('./elements/Todos')
+const Dones = require('./elements/Dones')
+const NewTodo = require('./elements/NewTodo')
+
+const TodoActions = require('./reducers/TodoActions')
+
+const html = Tram.html({
+  NewTodo,
+  Todos,
+  Dones
+})
+
+const home = (store, actions) => {
+  return html`
+    <div class="center">
+      <h1> Tram-One 🚋 </h1>
+      <h2> Todos App Example </h2>
+      <a href="/todo">TODO</a> | <a href="/results">RESULTS</a>
+    </div>
+  `
+}
+
+const todo = (store, actions) => {
+  const onUpdateInput = ({currentTarget: { value }}) => {
+    actions.updateInput(value)
+  }
+
+  const onAddTodo = () => {
+    actions.addTodo()
+  }
+
+  const onCompleteTodo = (index) => () => {
+    actions.completeTodo(index)
+  }
+
+  return html`
+    <div>
+      <div class="center">
+        <h1> Tram-One Todos 🚋 </h1>
+        <a href="/">HOME</a> | <a href="/results">RESULTS</a>
+      </div>
```

Comment:
to reduce how many times we write this, lets make a custom element which is the header with all the links in it (even if it a bit redundant)